### PR TITLE
Added Major Changes from CHANGELOG.md to Welcome Package ChangeLog

### PR DESCRIPTION
**Pull request by confused-Techie**
This adds a few entries that were missing from our Welcome Package ChangeLog View, in anticipation of these being listing in the release

### CODE REVIEW

File: packages/welcome/lib/changelog-view.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ export default class ChangeLogView {
           </div>
           <div className="welcome-panel">
             <p>Take a look at some of the awesome things {atom.branding.name} has changed:</p>
+            <p>Feel free to read our [Full Change Log](https://github.com/pulsar-edit/pulsar/blob/master/CHANGELOG.md).</p>
```
Comment (DeeDeeG) [inline packages/welcome/lib/changelog-view.js:62]: Can we do Markdown style links in this view? I guess I can test this locally real quick...

Comment (confused-Techie) [reply packages/welcome/lib/changelog-view.js]: Nope, don't think we can. Instinct took over, fixing now, and I'll test it real fast again. My bad

File: packages/welcome/lib/changelog-view.js [hunk 2]
```diff
@@ -82,7 +83,7 @@ export default class ChangeLogView {
               </li>
               <li>
                 <a href="#">
```
Comment (DeeDeeG) [inline packages/welcome/lib/changelog-view.js:85]: Not new in this PR, but this anchor element has no proper `href` target. It doesn't link to anywhere at the moment.

File: packages/welcome/lib/changelog-view.js [hunk 3]
```diff
@@ -82,7 +83,7 @@ export default class ChangeLogView {
...
-                  Migrated to a new Repository Backend
+                  Migrated to a new Pulsar Package Repository Backend
                 </a>
```
Comment (DeeDeeG) [inline packages/welcome/lib/changelog-view.js:87]: Not new in this PR, but this anchor element has no proper `href` target. It doesn't link to anywhere at the moment.

File: packages/welcome/lib/changelog-view.js [hunk 4]
```diff
@@ -50,7 +50,7 @@ export default class ChangeLogView {
       <div className="welcome">
         <div className="welcome-container">
           <div className="header">
-            <a title="pulsar-edit.dev" href={atom.branding.urlWeb}>
+            <a title="pulsar-edit.dev" href="https://github.com/pulsar-edit/pulsar/blob/master/CHANGELOG.md">
```
Comment (DeeDeeG) [inline packages/welcome/lib/changelog-view.js:53]:
```suggestion
            <a title="Full Changelog" href="https://github.com/pulsar-edit/pulsar/blob/master/CHANGELOG.md">
```

Or 

```suggestion
            <a href="https://github.com/pulsar-edit/pulsar/blob/master/CHANGELOG.md">
```

???

File: packages/welcome/lib/changelog-view.js [hunk 5]
```diff
@@ -81,45 +82,55 @@ export default class ChangeLogView {
...
-                <a href="#">
+                <a href="https://github.com/pulsar-edit/settings-view/pull/2">
                   Better error messages when a package fails to install.
                 </a>
               </li>
               <li>
-                <a href="#">
+                <a href="https://github.com/pulsar-edit/pulsar/pull/67">
                   Configuration file watching fixes
                 </a>
```
Comment (DeeDeeG) [inline packages/welcome/lib/changelog-view.js:97]: So, this links to the same PR https://github.com/pulsar-edit/pulsar/pull/67 as another changelog entry, "Removed experimental file watchers on the editor".

But maybe this is two separate things of note that happen to come from one PR. I leave it to your judgment if this is something to remove as duplicate, or note both things separately out of one PR.

Comment (Daeraxa) [reply packages/welcome/lib/changelog-view.js:97]: I think initially we were being pretty bad about broad scoped PRs, not something that seems to happen now though.

Comment (confused-Techie) [reply packages/welcome/lib/changelog-view.js]: Good catch, I've just combined the entries to leave one entry for both changes. This wasn't intentional so seems best to combine them

**Review by DeeDeeG — APPROVED** [review]
Confirmed all the links are what they say they are, can't find any more stuff to comment on/that needs anything.

Thanks for working through all these tiny feedbacks, looks good to me! :+1:

**Comment by confused-Techie** [comment]
Awesome, thanks @DeeDeeG!
I'll go ahead and merge this one

**Pull request merged**